scripts/signing/common.py

```diff
@@ -0,0 +1,208 @@
+import hashlib
+import os
+import tempfile
+import time
+
+import requests
+
+
+ARTIFACTORY_URL = "https://splunk.jfrog.io/splunk"
+ARTIFACTORY_API_URL = f"{ARTIFACTORY_URL}/api"
+CHAPERONE_API_URL = "https://chaperone.re.splunkdev.com/api-service"
+SIGNED_ARTIFACTS_REPO_URL = f"https://repo.splunk.com/artifactory/signed-artifacts"
+STAGING_URL = "https://repo.splunk.com/artifactory"
+STAGING_REPO = "signalfx-agent-local"
+STAGING_REPO_URL = f"{STAGING_URL}/{STAGING_REPO}"
+SIGN_TYPES = ("GPG", "RPM", "WIN")
+DEFAULT_ARTIFACTORY_USERNAME = "signalfx-agent"
+DEFAULT_STAGING_USERNAME = "srv-signalfx-agent"
+DEFAULT_TIMEOUT = 300
+
+
+def upload_file_to_artifactory(src, dest, user, token):
+    print(f"uploading {src} to {dest} ...")
+    with open(src, "rb") as fd:
+        data = fd.read()
+        headers = {"X-Checksum-MD5": hashlib.md5(data).hexdigest()}
+        resp = requests.put(dest, auth=(user, token), headers=headers, data=data)
+        assert resp.status_code == 201, f"upload failed:\n{resp.reason}"
+        return resp
+
+
+def submit_signing_request(src, sign_type, token):
+    print(f"submitting '{sign_type}' signing request for {src} ...")
```

Comment:
Add some blank lines to split up the various "paragraphs" of code.  This is just one big wall of code and it's harder to follow like this.

Reply:
This goes for most of the Python code.